packages/paste-website/src/pages/patterns/copy-to-clipboard/index.mdx

```diff
@@ -0,0 +1,454 @@
+---
+title: Copy to clipboard
+description: Give customers guidance on what steps to take if content is either unavailable or can't be displayed.
```

Comment:
This is the empty state description still :P. How about `A library that allows the user to copy text with the click of a button.`

Reply:
if you think i'm some lazy engineer and would just copypasta the empty state pattern like that, you'd be totally right

Reply:
actually i'm going to remove these patterns pages for now; publishing just the library and waiting for the actual component makes more sense for the Patterns pages i think